conf/broker.conf

```diff
@@ -960,8 +960,8 @@ managedLedgerCacheCopyEntries=false
 # Threshold to which bring down the cache level when eviction is triggered
 managedLedgerCacheEvictionWatermark=0.9
 
-# Configure the cache eviction frequency for the managed ledger cache (evictions/sec)
-managedLedgerCacheEvictionFrequency=100.0
+# Configure the cache eviction interval for the managed ledger cache
+managedLedgerCacheEvictionIntervalMs=10
```

Comment:
The PR introduced a new configuration and deprecate an old configuration, it's better to discuss in the dev email list first.

Reply:
Ok, will send a discuss email to dev mail list